vdev/src/commands/check/fmt.rs

```diff
@@ -0,0 +1,14 @@
+use anyhow::Result;
+
+use crate::app;
+
+/// Check that all files are formatted properly
+#[derive(clap::Args, Debug)]
+#[command()]
+pub struct Cli {}
+
+impl Cli {
+    pub fn exec(self) -> Result<()> {
+        app::exec_app_path("cargo", ["fmt", "--", "--check"])
+    }
+}
```

Comment:
Does it actually make sense to have this function? In the `script/fmt.sh` we directly call `cargo fmt` to do the formatting, why don't we do the same for checks?